pkg/common/configs/configvalidator_test.go

```diff
@@ -2012,3 +2012,227 @@ func TestCheckQueuesStructure(t *testing.T) {
 		})
 	}
 }
+
+func TestCheckQueues(t *testing.T) {
+	testCases := []struct {
+		name             string
+		queue            *QueueConfig
+		level            int
+		errorExpected    bool
+		expectedErrorMsg string
+		validateFunc     func(t *testing.T, queue *QueueConfig)
+	}{
+		{
+			name: "Invalid ACL Format for AdminACL",
+			queue: &QueueConfig{
+				Name:      "validQueue",
+				AdminACL:  "admin group extra",
+				SubmitACL: "submit",
+				Queues:    []QueueConfig{{Name: "validSubQueue"}},
+			},
+			level:            0,
+			errorExpected:    true,
+			expectedErrorMsg: "multiple spaces found in ACL: 'admin group extra'",
+		},
+		{
+			name: "Invalid ACL Format for SubmitACL",
+			queue: &QueueConfig{
+				Name:      "validQueue",
+				AdminACL:  "admin",
+				SubmitACL: "submit group extra",
+				Queues:    []QueueConfig{{Name: "validSubQueue"}},
+			},
+			level:            0,
+			errorExpected:    true,
+			expectedErrorMsg: "multiple spaces found in ACL: 'submit group extra'",
+		},
+		{
+			name: "Duplicate Child Queue Names",
+			queue: &QueueConfig{
+				Name:      "root",
+				AdminACL:  "admin",
+				SubmitACL: "submit",
+				Queues: []QueueConfig{
+					{Name: "duplicateQueue"},
+					{Name: "duplicateQueue"},
+				},
+			},
+			level:            0,
+			errorExpected:    true,
+			expectedErrorMsg: "duplicate child name found with name duplicateQueue, level 0",
```

Comment:
Let's put duplicateQueue in single quotes for consistency: `duplicate child name found with name 'duplicateQueue', level 0`

Reply:
Something similar here, the expectedErrorMsg need to match the return error when it call the function `checkQueues`, the error message to duplicate name is [here](https://github.com/apache/yunikorn-core/blob/master/pkg/common/configs/configvalidator.go#L660):
```
if queueMap[strings.ToLower(child.Name)] {
    return fmt.Errorf("duplicate child name found with name %s, level %d", child.Name, level)
}
```
Do I need to add single quote `'` to the error messages for both?

Reply:
Update the code.